packages/modules/vehicles/polestar/auth.py

```diff
@@ -177,7 +176,7 @@ def _get_auth_code(self) -> Optional[str]:
         log.info("_get_auth_code:attempting to get new code")
         try:
             result = self.client_session.post(
-                BASE_URL+f"/as/{self.resume_path}/resume/as/authorization.ping",
+                BASE_URL + f"{self.resume_path}",
```

Comment:
```suggestion
               f"{BASE_URL}{self.resume_path}",
```
Hier vermischt Du zwei Darstellungsweisen eines Strings.
Oder die ältere Variante:
```suggestion
                BASE_URL + self.resume_path,
```